backend-agent/main.py

```diff
@@ -40,6 +42,15 @@
              } if langfuse_handler else {
                  'callbacks': [status_callback_handler]}
 
+# Set up the upload folder dynamically
+UPLOAD_FOLDER = './uploads'  # You can change this to a different path if needed
```

Comment:
Let's use the `.env` file for setting `UPLOAD_FOLDER`, while keeping `./uploads` as default choice in case it is not set.

Moreover, it's probably better to name the variable `DASHBOARD_DATA` or something similar (in future, the idea is that scores will be computed with scans and stored here and upload will be an option but not mandatory)